libraries/libfc/include/fc/container/ordered_diff.hpp

```diff
@@ -37,50 +42,50 @@ class ordered_diff {
       diff_result result;
       while (s < source.size() || t < target.size()) {
          if (s < source.size() && t < target.size()) {
+            FC_ASSERT(s < std::numeric_limits<SizeType>::max());
+            FC_ASSERT(t < std::numeric_limits<SizeType>::max());
```

Comment:
Please move these two *before* the `if`, otherwise we skip the increments at line 84 and 89.